contributors/devel/pull-requests.md

```diff
@@ -81,21 +79,24 @@ For PRs with a release note:
     Your release note here
```

Comment:
Suggest that we add some more guidance here. We don't really have a release note template, just spaces for release notes in PR templates. Let's try at a minimum:
"Addresses issue #(link to issue, to feature description, or to any other document that describes what the PR is a part of). (Add more details about specific work of PR)

Reply:
Good suggestion! 
One concern is, do we want to keep two copies of the release note template text both in PR template and here? They may be different as time goes.
And note that we have at least four places wrt release note stuff: prow plugin code, pr template, relnotes tool and here. It's hard to guarantee that they keep consistent when the plugin code changes. cc @cjwagner

Reply:
>One concern is, do we want to keep two copies of the release note template text both in PR template and here? They may be different as time goes. [people may forget to update the other one when they update one place]

anyway, it might be good to list the release note template in both the doc and PR template, because people may only read one of them.

Reply:
>"Addresses issue #(link to issue, to feature description, or to any other document that describes what the PR is a part of). (Add more details about specific work of PR)"

This will lead to including an issue in each release note. Do we really need this? (since we will link to the PR in our final release note)

Reply:
Also keep in mind that the approval-handler already enforces requiring an associated issue on PRs and will tell users to add one when one is needed.

Reply:
Yeah that's one point I was thinking of. Since we will link to the associated PR for each release note in the final CHANGELOG, and the PR already has an associated issue in it's PR body, so I don't think it's so necessarily to include associated issue in release note. @Bradamant3 WDYT?